gravitee-apim-console-webui/src/components/gio-ee-unlock-dialog/gio-ee-unlock-dialog.component.html

```diff
@@ -34,14 +34,7 @@
   ></h2>
   <div class="gio-ee-unlock-dialog__content__description" [innerHTML]="featureMoreInformation.description | safe : 'html'"></div>
   <div class="gio-ee-unlock-dialog__content__action">
-    <a
-      class="gio-ee-unlock-dialog__content__action__button"
-      mat-button
-      mat-stroked-button
-      color="accent"
-      target="_blank"
-      [href]="licenseService.ctaLinkPrefix + featureMoreInformation.utmTag"
-    >
+    <a class="gio-ee-unlock-dialog__content__action__button" mat-button mat-stroked-button color="accent" target="_blank" [href]="trialURL">
```

Comment:
not sure you need to add mat-button and mat-stroked-button. You can test it but mat-stroked-button should be enough I think